### PR TITLE
Fixing threshold for Log Processor input queue

### PR DESCRIPTION
**Pull request by kostaspap**
## Background

Adjusting threshold for Log processor input SQS queue

### CODE REVIEW

**Review by austinbyers — APPROVED** [review]

**Review by rleighton — APPROVED** [review]

File: internal/core/custom_resources/resources/alarms_sqs.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func putSQSAlarmGroup(props SQSAlarmProperties) error {
 			fmt.Sprintf("Panther-%s-%s", sqsAgeAlarm, props.QueueName))
 		input.MetricName = aws.String("ApproximateAgeOfOldestMessage")
 		input.Statistic = aws.String(cloudwatch.StatisticMaximum)
-		var threshold float64 = 900.0
+		threshold := 900.0
```
Comment (alxarch) [inline internal/core/custom_resources/resources/alarms_sqs.go:93]: I would prefer to change this to `(15*time.Minute).Seconds()` for clarity

Comment (alxarch) [reply internal/core/custom_resources/resources/alarms_sqs.go:93]: Why does this const differ from the one defined in the YAML file?

Comment (kostaspap) [reply internal/core/custom_resources/resources/alarms_sqs.go:93]: This is the default value. The value in the CFN template is override for a specific queue.

Comment (kostaspap) [reply internal/core/custom_resources/resources/alarms_sqs.go]: Updated.

**Pull request merged**